src/docs/content/disaster-recovery/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Disaster Recovery"
+date: 2021-09-02 T16:04:15-06:00
+draft: false
+weight: 510
+showChildren: true
+---
+
+## Introduction
+
+In this section we will cover the disaster recovery solution for MuShop deployed on OCI (Oracle Cloud Infrastructure).
+We will use the recently announced [Cross-Region Autonomous Data Guard for Autonomous databases](https://blogs.oracle.com/datawarehousing/cross-region-autonomous-data-guard-your-complete-autonomous-database-disaster-recovery-solution) feature to demonstrate the DR (Disaster Recovery) solution.
+
+![disaster-recovery](../observability/images/disaster-recovery.png)
```

Comment:
The image indicates MuShop with "Prod" and "DR" regions denoted. Might "Primary" be more appropriate?